core_test.go

```diff
@@ -1182,6 +1183,33 @@ func TestImageToMatRGB(t *testing.T) {
 	}
 }
 
+func TestImageToMatGray(t *testing.T) {
+	file, err := os.Open("images/gocvlogo.jpg")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer file.Close()
+	imgSrc, _, err := image.Decode(file)
+	if err != nil {
+		log.Fatal(err)
+	}
+	img0 := image.NewGray(imgSrc.Bounds())
+	draw.Draw(img0, imgSrc.Bounds(), imgSrc, image.ZP, draw.Src)
+
+	mat, err := ImageToMatRGB(img0)
```

Comment:
Shouldn't this be `ImageToMatGray()` here?

Reply:
Facepalm to me =)